src/components/Newsletter.jsx

```diff
@@ -31,8 +31,8 @@ export function Newsletter() {
             Awards
           </h3>
           <div className="my-4 flex flex-wrap items-center justify-evenly gap-4">
-            <Image src={CommunityAward} width="500" className="rounded" />
-            <Image src={TeacherAward} width="500" className="rounded" />
+            <Image alt={"Community Award"} src={CommunityAward} width="500" className="rounded" />
```

Comment:
Can we have a better alt text? For example: "Image Showcasing the GitHub Stars Nova 2021 Award, awarded to Eddie Jaoude, Founder of EddieHub"

Reply:
Thanks for the suggestion, I'll update the code